lib/mongoose-history.js

```diff
@@ -113,12 +113,12 @@ module.exports = function historyPlugin(schema, options) {
   });
 
   function createHistoryDoc(d, operation) {
-    d.__v = undefined;
+    const { __v, ...doc } = d;
```

Comment:
Instead of setting `__v` to undefined I'm using the object spread operator to copy all of the other fields into a variable called `doc` which I'm then setting on the history document.